src/ch/ch_monitor.c

```diff
@@ -653,7 +653,7 @@ virCHMonitorPutNoContent(virCHMonitor *mon, const char *endpoint)
     VIR_LOCK_GUARD lock = virObjectLockGuard(mon);
     g_autofree char *url = NULL;
     int responseCode = 0;
-    int ret = -1;
+    int ret = 0;
```

Comment:
Could you please add some detail on why this change is required in the commit message?